src/diffpy/structure/atom.py

```diff
@@ -167,16 +204,41 @@ def msdCart(self, vc):
 
 
     def __repr__(self):
-        """simple string representation"""
+        """
+        String representation of the atom's ``atype``, ``xyz``, and ``occupancy``.
+
+        Parameters
+        ----------
+        None.
+
+        Returns
+        -------
+        s : str
+            atom's ``atype``, ``xyz``, and ``occupancy``.
+        """
+
         xyz = self.xyz
         s = "%-4s %8.6f %8.6f %8.6f %6.4f" % \
                 (self.element, xyz[0], xyz[1], xyz[2], self.occupancy)
         return s
 
 
     def __copy__(self, target=None):
-        """Return a copy of this instance.
         """
+        Create a copy of this instance.
+
+        Parameters
+        ----------
+        Target : Atom or None
```

Comment:
Use the same capitalization, i.e., `target`.  When function is defined with some `argument=None` by convention it means that that argument is optional.  Please correct to:
```
target : Atom, optional
    An already existing Atom object to be updated to a duplicate
    of this Atom.  Create a new Atom object when not specified.
    This argument facilitates extension of the `__copy__` method
    in a derived class.    
```